package/MDAnalysis/analysis/psa.py

```diff
@@ -591,6 +595,30 @@ def dist_mat_to_vec(N, i, j):
         raise ValueError(err_str)
 
 
+def validate_dist_mat_to_vec_inputs(N, i, j):
+    """
+    Validate the inputs of the function dist_mat_to_vec else raise an exception
+    :param N: int
+    :param i: int
+    :param j: int
+    :return: null
```

Comment:
We follow numpy style docs in new code. You can have a look into other files for the look.